updates for handling forcing outputs in ladas cycling

saraqzhang opened the pull request:
modified:   src/Applications/GEOSdas_App/GEOSdas.csm
	modified:   src/Applications/GEOSdas_App/fvpsas

saraqzhang committed:
updates for handling forcing outputs in ladas cycling
	modified:   src/Applications/GEOSdas_App/GEOSdas.csm
	modified:   src/Applications/GEOSdas_App/fvpsas

gmao-rreichle commented on src/Applications/GEOSdas_App/GEOSdas.csm:
```diff
@@ -5866,10 +5867,10 @@ endif
-               mkdir -p              $FVHOME/recycle/holdforc
-               /bin/cp  *2d_lfo*nc4  $FVHOME/recycle/holdforc/
-            endif
+            #if ( $LDAS_ANA ) then
```
Is the if condition here intentionally commented out?   Based on a quick look it seems that the if condition should be there, but I might be wrong.

saraqzhang replied:
@gmao-rreichle ,   that block was leftover from before, I forgot to delete it in the update. I will do it now the push the change.  thanks for spotting it.

saraqzhang committed:
delete the redundant foring hold

gmao-rreichle commented:
Looks ok to me, but I'm not the judge here.
I changed the label to "0 diff trivial" because the edits are all within `if ( $LDAS_ANA )` blocks.
@rtodling, from what I understand, this fix is needed because recent ADAS changes broke the weakly-coupled land assimilation.

mathomp4 committed:
fixed typo for zeit_c?.x MovePredictorOutput (fvpsas)

mathomp4 commented:
The force push above was:
```
gh repo clone GEOS-ESM/GEOSadas GEOSadas-Reset
cd GEOSadas-Reset
git checkout feature/saraqzhang/ladasupdate
git reset HEAD~5
git add .
git commit -m "fixed typo for zeit_c?.x MovePredictorOutput (fvpsas)"
git push --force origin feature/saraqzhang/ladasupdate
```
The add was for the "fix misspelling" commit that was previously there.

saraqzhang committed:
update fvsetup prompts and defaults for ladas
    add a sample ladas input file to testsuites

saraqzhang commented:
@gmao-rreichle @rtodling     
I updated the prompts and defaults concerning LDAS coupling in fvsetup and added the sample input file for ladas setup into testsuites/

gmao-rreichle committed:
fixed alignment, white-space change only (GEOSdas.csm)

saraqzhang commented:
the changes here are consolidated into PR #275.
